src/vs/base/browser/ui/findinput/findInput.css

```diff
@@ -36,10 +36,12 @@
 	animation: monaco-findInput-highlight-1 100ms linear 0s;
 }
 .hc-black .monaco-findInput.highlight-0 .controls,
```

Comment:
Fixed so that hc-light uses the light colors/animations. Thansk for catching that.

As for the hard coded bits, are you referencing the disabled background rules and the animation backgrounds? I'd need to go decide on specific colors for each theme if all of them have been using the same values all this time. Might make sense to do this on a separate issue since it goes beyond HC.